x-pack/plugins/apm/public/store/urlParams.ts

```diff
@@ -148,8 +152,10 @@ export const getUrlParams = createSelector(
 );
 
 export interface IUrlParams {
+  activeTimelineId: string | null;
```

Comment:
Can we name this `waterfallItemId` to stay with the waterfall terminology? I also think "active" is superfluous (unless we also plan to store the inactive waterfall ids in the url).

Reply:
In the UI it's referred to as "timeline" so I went with that since it shows up in the URL and could _conceivably_ be manipulated by users. I guess they never will so it probably doesn't matter. Wish those names hadn't diverged :/ whether active is there or not is definitely not a hill I'm going to die on either :)